en/micro-integrator/docs/references/connectors/configure-connector-exporter-project.md

```diff
@@ -0,0 +1,27 @@
+## Configure the connector in WSO2 Integration Studio
+
+1. Open WSO2 Integration Studio and create an ESB Solution Project.
+<p><img src="../../assets/img/connectors/solution-project.png" title="Creating a new ESB Solution Project" width="800" alt="Creating a new ESB Solution Project" /></p>
+
+2. Right click on the project that you created and click on 'Add or Remove Connector' -> Add Connector.
```

Comment:
```suggestion
2. Right click on the project that you created and click on **Add or Remove Connector** -> **Add Connector**.
```

Let's bold all terms that come directly from the UI.

Reply:
Done